lib/graphql/execution/executor.ex

```diff
@@ -182,4 +182,23 @@ defmodule GraphQL.Execution.Executor do
   defp field_entry_key(field) do
     Map.get(field, :alias, field.name)
   end
+
+  defp collect_fragment(context, runtime_type, selection, field_fragment_map) do
+    # TODO: should_include?(context, selection.directives)
```

Comment:
Assuming this is a note for next directive PR? Remove it once implemented please.